docs/bigquery/snippets.py

```diff
@@ -2153,22 +2153,28 @@ def test_undelete_table(client, to_delete):
 
```

Comment:
It won't let me comment on the lines above since they're not part of the diff, but update the first part of the sample to the new TODO format:
```
    # TODO(developer): Uncomment the lines below and replace with your values.
    # from google.cloud import bigquery
    # import time
    # client = bigquery.Client()
    # dataset_id = 'my_dataset'  # replace with your dataset ID
    # table_id = 'my_table'      # replace with your table ID
```